tests/wrappers/time_limit_wrappers_test.py

```diff
@@ -81,5 +81,21 @@ def test_time_limit_fork(env: LlvmEnv):
         fkd.close()
 
 
+# @pytest.mark.xfail(strict=True, reason="https://github.com/facebookresearch/CompilerGym/issues/648")
+def test_time_limit(env: LlvmEnv):
+    """Check CycleOverBenchmarks does not break TimeLimit"""
+    env = TimeLimit(env, max_episode_steps=1)
+    env = CycleOverBenchmarks(
+        env,
+        benchmarks=[
+            "benchmark://cbench-v1/crc32",
+        ],
+    )
+    env.reset()
+    _, _, done, _ = env.step(0)
+
+    assert done
```

Comment:
```suggestion
    assert done
    assert info["TimeLimit.truncated"], info
```

Best to also test the reason why `done=True`.